api/src/pcapi/utils/requests.py

```diff
@@ -39,16 +39,16 @@ def _redact_url(url: str) -> str:
     return re.sub("api_token=[^&^$]+", "api_token=[REDACTED]", url)
 
 
-def _wrapper(request_func: Callable, method: str, url: str, log_at_error_level=True, **kwargs: Any) -> Response:  # type: ignore [no-untyped-def]
+def _wrapper(request_func: Callable, method: str, url: str, log_at_warning_level=True, **kwargs: Any) -> Response:  # type: ignore [no-untyped-def]
     timeout = kwargs.pop("timeout", REQUEST_TIMEOUT_IN_SECOND)
     try:
         response = request_func(method=method, url=url, timeout=timeout, **kwargs)
     except Exception as exc:
         # If the URL is not controlled by us, there is probably no
         # reason to send the error to Sentry. Logging an INFO message
         # is enough if we ever need to debug.
-        if log_at_error_level:
-            logger_method = logger.exception
+        if log_at_warning_level:
+            logger_method = logger.warning
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Est ce que le commentaire est toujours d'actualité ici ? Pourquoi ne pas tout push en `info`?
‏

Reply:
j'enlève le commentaire ! je garde le niveau warning car c'est adapapté je trouve